src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -175,25 +194,37 @@ public static FormulaValue Text(EvalVisitor runner, EvalVisitorContext context,
                 }
             }
 
+            // We limit the format string size to 100 chars
+            if (formatString != null && formatString.Length > 100)
```

Comment:
>100 [](http://example.com/codeflow?start=62&length=3)

this should be a const field. 
Then we don't have to repeat the literal on line 201.